src/util/validation/metadata.ts

```diff
@@ -0,0 +1,53 @@
+import { z } from 'zod'
+
+import { valid } from './zod'
+
+const groupMetadataSchema = z.object({
+  name: valid.name,
+  description: valid.description.optional(),
+  updatedAt: z.string(),
+  forumLink: valid.url.optional(),
+  other: z.string().optional(),
+})
+export type UIGroupMetadata = z.infer<typeof groupMetadataSchema>
+
+function isGroupMetadata(metadata: unknown): metadata is UIGroupMetadata {
+  return groupMetadataSchema.safeParse(metadata).success
```

Comment:
`parse` and `safeParse` are zod's rough equivalent to the AJV `validate` methods (... I think)